datastore/migrate/data/migrations/1707380318_add_columns.down.sql

```diff
@@ -0,0 +1,11 @@
+ALTER TABLE time_series
+	DROP COLUMN parameter_name,
+	DROP COLUMN function,
+	DROP COLUMN period,
+	DROP COLUMN level,
+	DROP CONSTRAINT unique_main,
```

Comment:
I changed my mind, it is probably a good idea to have guards in the "down" script, in case of a partial run of "up" script.